SPECS-EXTENDED/python-curio/python-curio.spec

```
@@ -29,11 +29,11 @@ many of its ideas have influenced other libraries with similar functionality.
 Summary:        Building blocks for performing concurrent I/O
 Name:           python-%{pkgname}
 Version:        1.4
-Release:        3%{?dist}
+Release:        4%{?dist}
 License:        BSD
 URL:            https://github.com/dabeaz/curio
 #Source0:       https://files.pythonhosted.org/packages/source/c/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
-Source0:        https://files.pythonhosted.org/packages/source/c/%{pypi_name}/%{name}-%{version}.tar.gz
+Source0:        https://files.pythonhosted.org/packages/73/f4/9d50359189a255e94c0b66eb6c3a29fba57571bbc80049c365e269b10c4e/curio-%{version}.tar.gz
```

You've changed the name of the source tarball, so the spec will now fail to pack (the tools will look for `curio-1.4.tar.gz`):
```suggestion
Source0:        https://files.pythonhosted.org/packages/73/f4/9d50359189a255e94c0b66eb6c3a29fba57571bbc80049c365e269b10c4e/curio-%{version}.tar.gz#/%{name}-%{version}.tar.gz
```